### PR TITLE
Update steering members

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/steering/pull/157

/hold
until https://github.com/kubernetes/steering/pull/157 has merged

### CODE REVIEW

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -10,5 +10,10 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
-bob
-alice
```
Comment (cblecker) [inline SECURITY_CONTACTS]: Why this change?

Comment (nikhita) [reply SECURITY_CONTACTS]: The old file contained dummy names (bob and alice). I figured I'd update this file while we are cleaning up OWNERS, but happy to move it to another PR too.

**Comment by nikhita** [comment]
@cblecker will leave for you to cancel the hold as per https://github.com/kubernetes/kubernetes-template-project/pull/33#discussion_r411515183

File: SECURITY_CONTACTS (hunk 1 above)
Comment (cblecker) [reply SECURITY_CONTACTS]: I think the dummy names were intentional, as this *shouldn't* be steering in the template

Comment (nikhita) [reply SECURITY_CONTACTS]: Derp, fixed! Too many repos and too many changes lol
